src/components/EditableText.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { createRef, Fragment, Component } from "react";
+
+type InputState = {
+  initialValue: string;
+  value: string;
+  edit: boolean;
+};
+
+type Props = {
+  value: string;
+  updateName: (name: string) => void;
```

Comment:
Maybe more generic name like `onChange` would be better here

Reply:
Agreed

Reply:
> Could you add a screenshot / small video / description of what you implemented by any chance?

Sure, but can you try it here in the meantime https://2h6i2.csb.app
Just to the top left